bin/dm_mripicture.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+"""
+Creates MRI axial pictures for custom T-shirt.
+
+Usage:
+    mripicture.py [options] <study>
+    mripicture.py [options] <study> <participant>
+    mripicture.py [options] <study> <participant> <output>
```

Comment:
I think adding the ellipsis will allow `participant` to be a repeating argument. Please double check this.

```suggestion
    mripicture.py [options] <study> <participant>...
    mripicture.py [options] <study> <participant>... <output>
```

Reply:
i would maybe just let `participant` be a repeatable `[option]`. It's much easier to use unix pipes that way (i.e `xargs` pipes) and it simplifies the user interface. 

Such as:
```
mripicture.py [options] [-s <SUBJECT>]... <study>
```

Same goes for `output`, if it's optional, then specify it as an optional argument rather than positional